crates/matrix-sdk/src/client/builder.rs

```diff
@@ -301,6 +303,12 @@ impl ClientBuilder {
         self
     }
 
+    /// Public for test only
```

Comment:
Let's add `#[doc(hidden)] ` for this.

Reply:
Can't we use `#[cfg(test)]`? Is it used by another crate?

Reply:
Yeah, I'm using it in crates/matrix-sdk-ui/src/timeline/sliding_sync_ext.rs